configure

```diff
@@ -493,34 +520,42 @@ install_local_clouseau() {
 	    _LOGBACK_CLASSIC_JAR=logback-classic-"$LOGBACK_VERSION".jar
 	    _LOGBACK_CLASSIC_URL="$_LOGBACK_DIST_URL"/logback-classic/"$LOGBACK_VERSION"/"$_LOGBACK_CLASSIC_JAR"
 
+	    echo "==> Installing local Clouseau (dist method)..."
 	    check_local_clouseau_dir "$CLOUSEAU_DIR"
+	    echo "Creating Clouseau directory at ${CLOUSEAU_DIR}..."
 	    mkdir -p "$CLOUSEAU_DIR"
 
 	    echo "Fetching Clouseau from $_DIST_URL..."
 
 	    fetch_file clouseau.zip "$_DIST_URL"
-
+	    
+	    echo "Extracting Clouseau distribution..."
 	    if ! unzip -q -j clouseau.zip -d "$CLOUSEAU_DIR"; then
 		printf "ERROR: Clouseau distribution package (clouseau.zip) could not be extracted.\n" >&2
 		exit 1
 	    fi
 
+	    echo "Removing temporary Clouseau zip file..."
 	    rm clouseau.zip
 
 	    echo "Fetching Logback $LOGBACK_VERSION from $_LOGBACK_DIST_URL..."
-
 	    fetch_file "$CLOUSEAU_DIR"/"$_LOGBACK_CORE_JAR" "$_LOGBACK_CORE_URL"
+	    echo "Fetching Logback $LOGBACK_VERSION classic from $_LOGBACK_CLASSIC_URL..."
```

Comment:
The messages about fetching files could made part of the `fetch_file` shell functions instead.  Also, this extra `echo` seems redundant since a similar message is print before calling `fetch_file`.